lib/Login/SocialLogin.class.inc

```diff
@@ -1,4 +1,35 @@
 <?php
+// Autoloader for the new Facebook API
+require_once(LIB . 'External/Facebook/autoload.php');
```

Comment:
This would be better in the `getFacebookObj` function as it was before, it avoids the need for loading/using the facebook classes when it's not need or is disabled due to not having facebook configs set (also as a general thing it keeps the behaviour as similar as possible, to avoid weird things)